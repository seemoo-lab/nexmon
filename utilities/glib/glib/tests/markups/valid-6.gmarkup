<!DOCTYPE foo "foo" [
<!ELEMENT foo ANY >
]>
<foo></foo>
