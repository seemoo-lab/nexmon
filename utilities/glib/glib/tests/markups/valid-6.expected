PASS '<!DOCTYPE foo "foo" [
<!ELEMENT foo ANY >
]>'
ELEMENT 'foo'
  TEXT ''
END 'foo'
